docs/UserGuide.md

```diff
@@ -10,11 +10,8 @@ The app is optimized for use via a Command Line Interface (CLI) while still havi
 benefits of a Graphical User Interface (GUI). If you can type fast, CHIM can get your management tasks done
 faster than traditional GUI apps.
 
-* Table of Content
-  - [Quick start](#quick-start)
```

Comment:
Any reason to remove?

Reply:
I wanted to use the same autogenerated TOC as DG